Adding limit reference for permissions grants collections (#2027)
MicrosoftDocs/entra-docs: docs/identity/enterprise-apps/manage-app-consent-policies.md

> Deleted app consent policies can't be restored. If you accidentally delete a custom app consent policy, you need to re-create the policy.

## Permission grant policy limits

> A Microsoft Entra tenant enforces limits on the number of IDs that can be defined within the collections of all permissionGrantConditionSet objects.
> For each collection type listed below, the combined total number of values across all permission grant policies and condition sets in the tenant cannot exceed 100 entries.

Reviewer: 🔴 **Self-contained rewrite (replaces lines 265-266).** This fixes several issues at once: removes the dangling "listed below" (no list follows), bridges the "app consent policy" ↔ "permission grant policy" terminology used in the rest of the article, code-formats the Graph resource type, and uses the contraction per Microsoft style.

```suggestion
App consent policies are implemented as permission grant policies in Microsoft Graph. A Microsoft Entra tenant enforces limits on the number of IDs defined within the collections of all `permissionGrantConditionSet` objects. For each collection type, the combined total number of values across all permission grant policies and condition sets in the tenant can't exceed 100 entries.
```

> A tenant can contain multiple permission grant policies and multiple condition sets within each policy. However, the combined number of entries for a specific collection type cannot exceed 100.

Reviewer: 🟢 **Use the contraction** per Microsoft style ("cannot" -> "can't").

```suggestion
A tenant can contain multiple permission grant policies and multiple condition sets within each policy. However, the combined number of entries for a specific collection type can't exceed 100.
```

For example:

Policy A contains 40 permission IDs in the permissions collection.
Policy B contains 35 permission IDs in the permissions collection.
Policy C contains 25 permission IDs in the permissions collection.

> The total number of permission IDs across all policies is 100.
> Adding another permission ID to any permissions collection in the tenant would exceed the supported limit and fail.

Reviewer: 🟢 **Make the concluding paragraph explicit.** These two sentences already render together as one paragraph; join them with a single space so the intent is clear.

```suggestion
The total number of permission IDs across all policies is 100. Adding another permission ID to any permissions collection in the tenant would exceed the supported limit and fail.
```
